buildworker.rst

```diff
@@ -82,40 +82,40 @@ For Mac:
     * Log in as the buildbot user.
     * Install either the Python 2.7 bundle from python.org [#]_, or pip.
     * Open a terminal window.
-    * Execute ``pip install buildbot-slave``.
+    * Execute ``pip install buildbot-worker``.
 
 For Windows:
 
     * Create a buildbot user as a "standard" user.
     * Install the latest version of Python 2.7 from python.org.
```

Comment:
This no longer needs to specify 2.7; we can use any version we want now :)

Reply:
This series of instructions was a tad bit confusing so I specified that the last step is only necessary if using pip.

Reply:
I'm a little lost on your comment here.  My comment was just that we can remove "2.7" from this line :)